app/src/main/java/com/mealsmadeeasy/data/FakeMealStore.kt

```diff
@@ -143,4 +143,25 @@ class FakeMealStore : MealStore {
 
         return Single.just(listOf(tacos, iceCream))
     }
+
+    override fun findMealById(id: String): Meal {
```

Comment:
Can we make this return a `Single<Meal>`? It'll make it a lot easier when we go to link this to the server.